x/group/module/abci.go

```diff
@@ -0,0 +1,17 @@
+package module
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/x/group/keeper"
+)
+
+func EndBlocker(ctx sdk.Context, k keeper.Keeper) {
```

Comment:
let's also add a test in abci_test, which calls `Endblocker()`, and check for the same things